wandb/wandb_run.py

```diff
@@ -131,10 +131,34 @@ def from_environment_or_defaults(cls, environment=None):
                   resume=resume)
         return run
 
+    def auto_project_name(self, api):
+        # if we're in git, set project name to git repo name + relative path within repo
+        root_dir = api.git.root_dir
+        if root_dir is None:
+            return None
+        repo_name = os.path.basename(root_dir)
```

Comment:
So if we aren’t in git there’s no magic?

Reply:
Yes for now.